### ISSUE TITLE
Cannot find source file 'linky-pipe.ts'

### ISSUE THREAD

**Issue by binarious** [dzonatan/angular2-linky#5]
While `karma test`:

```
WARNING in ./~/angular2-linky/linky-pipe.js
Cannot find source file 'linky-pipe.ts': Error: Can't resolve './linky-pipe.ts' in '/Users/MYUSER/projects/MYAPP/node_modules/angular2-linky'
```

Travis build of my project: https://travis-ci.org/Sirimangalo/meditation-plus-angular/builds/154833766#L2408

**Comment by dzonatan** [comment]
Ah. I forgot to include typescript files to package for source mapping. Fixed by v1.1.1.
Thanks! 🍺

**Comment by binarious** [comment]
Thanks!
